api/envoy/api/v2/core/base.proto

```diff
@@ -76,10 +77,43 @@ message Locality {
   string sub_zone = 3;
 }
 
+// BuildVersion combines SemVer version of extension with free-form build information
+// (i.e. 'alpha', 'private-build') as a set of strings.
+message BuildVersion {
+  // SemVer version of extension
+  type.SemanticVersion version = 1;
+
+  // Free-form build information i.e. 'alpha'
+  repeated string labels = 2;
+}
+
+// Version and identification for an Envoy extension.
+message Extension {
+  // This is the name of the Envoy filter as specified in the Envoy
+  // configuration, e.g. envoy.router, com.acme.widget.
+  string name = 1;
+
+  // Category of the extension
+  string category = 2;
+
+  // The version is a property of the extension and maintained independently
+  // of other extensions and the Envoy API.
+  BuildVersion version = 3;
+
+  // Indicates that the extension is present but was disabled via dynamic configuration.
+  bool disabled = 4;
```

Comment:
Naming nit: maybe should be "enabled"? I would assume most filters will be enabled (if they are programmed via dynamic configuration), so that visually "enabled = true" is easier to parse as something that is present and working (contrary to "disabled = false") in my opinion.

Reply:
This is a way to express that a filter is enabled by default. proto3 bool is false by default which is also indistinguishable from the field not being set at all. I thought it could less error prone to do it this way.

Reply:
I see you point, thanks for explaining! Should be good to leave as it is now.